apps/full-site-editing/full-site-editing-plugin/a8c-newspack-blocks/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { __ } from '@wordpress/i18n';
+
+// Change the block name through of the `registerBlockType` hook.
+addFilter( 'blocks.registerBlockType', 'a8c/renaming-newspack-blocks', ( settings, name ) =>
+	'newspack-blocks/homepage-articles' === name
+		? {
+			...settings,
+			title: __( 'Homepage Posts' ),
+			name: 'a8c/newspack-blocks',
```

Comment:
`/newspack-blocks` seems pretty generic for one block, shouldn't the name be something more specific?

Note also that we've been asked to use `jetpack/` prefix for .com blocks regardless of shipping method: paAmJe-Q1-p2#comment-2148

Reply:
Thanks, Mickael. Here is a comment about the block naming. https://github.com/Automattic/newspack-blocks/issues/168#issuecomment-558081139

Reply:
> /newspack-blocks seems pretty generic for one block, shouldn't the name be something more specific?

Yes, I was testing the approach. Going to update it with something more specific.

Reply:
>  Yes, I was testing the approach

Aight, sorry for the early comment. :-)

Reply:
It's completely fine. I've reopened this PR just in case. We need to change the category of the `Homepage Posts` block, so maybe it could be useful, yet.